internal/clients/kibana/alerting.go

```diff
@@ -20,11 +20,26 @@ func ruleResponseToModel(spaceID string, res *alerting.RuleResponseProperties) *
 
 	actions := []models.AlertingRuleAction{}
 	for _, action := range res.Actions {
-		actions = append(actions, models.AlertingRuleAction{
+
+		a := models.AlertingRuleAction{
 			Group:  action.Group,
 			ID:     action.Id,
 			Params: action.Params,
-		})
+		}
+
+		if alerting.IsNil(action.Frequency) {
+			a.Frequency = nil
+		} else {
```

Comment:
```suggestion
		if !alerting.IsNil(action.Frequency) {
```

The field will be nil by default no?